packages/neo-one-client/src/__tests__/provider/neoone/NEOONEDataProvider.test.js

```diff
@@ -0,0 +1,810 @@
+/* @flow */
+import { VM_STATE, common } from '@neo-one/client-core';
+import { AsyncIterableX } from 'ix/asynciterable/asynciterablex';
+import BigNumber from 'bignumber.js';
+
+import { toArray } from 'ix/asynciterable/toarray';
+
+import NEOONEDataProvider from '../../../provider/neoone/NEOONEDataProvider';
+
+import { keys, transactions } from '../../../__data__';
+
+function genExpectedInvocationResult(options: Object) {
```

Comment:
In some languages, e.g. php/hack, prefixing a method with `gen` indicates it's an async method. We don't follow that convention (and generally it's not done in JavaScript), but it'd be best to avoid using that prefix to avoid confusion. Instead, perhaps prefix with `create`. Same with all the other methods.